.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Related Issue
+- Issue goes here with URL/Number
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Checklist
+- [ ] Follow [Google developer documentation style guide](https://developers.google.com/style)
+- [ ] Issues must be Listed
```

Comment:
I would add an item here to "Check the deploy preview". It's one of the checks that runs automatically for each PR.

Reply:
Is there anything else I have to add in the checklist other than "check the deploy preview"

Reply:
I don't think the deploy preview is very intuitive.

Actually, we might be able to provide a template link:

https://deploy-preview-<pull-request-number>--elated-haibt-1b47ff.netlify.com/

How about the following as a set of simple instructions:

```
## Check the deployed preview

Each pull request is automatically staged. To review the deploy preview,
you can either click **Details** in the deploy preview check, or you can
visit the following link and substitute `<pull-request-number>` with the
number for this pull request:

https://deploy-preview-<pull-request-number>--elated-haibt-1b47ff.netlify.com/
```